common/check_injecting_rules

```diff
@@ -37,8 +38,9 @@ if __name__ == "__main__":
     try:
         registry = CollectorRegistry()
         session = get_session()
-        injecting_rules = 'select count(1) from {schema}rules where state=\'I\''.format(schema=schema)
-        result = session.execute(injecting_rules).fetchone()[0]
+        query = session.query(models.ReplicationRule.scope).filter(models.ReplicationRule.state.is_('I'))
```

Comment:
Better to import `RuleState` from `rucio.db.sqla.constants` and replace `'I'` with `RuleState.INJECT`.